contents/tutorials/holdout-testing.md

```diff
@@ -0,0 +1,78 @@
+---
+title: How to do holdout testing
+date: 2023-07-05
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-12.png
+tags: []
+---    
+
+Holdout testing is a type of [A/B testing](/docs/experiments) that keeps a "holdout" group for the long term after the completion of the main part of the experiment. This ensures the experiment doesn’t have negative long term effects or interaction with other experiments. This is particularly important in high-risk products or features like finance or your signup page.
```

Comment:
We should explain what holdout testing is without using the term "holdout" :) 
My suggestion needs to be reworked, but sharing the gist.

```suggestion
Holdout testing is a type of [A/B testing](/docs/experiments) to measure long term effects of product changes. In holdout testing, a small group of users is not shown your product changes for a long period of time, typically months (or even years!). This is contrast to regular A/B tests, which typically run for a few weeks

Holdout testing enables you to ensure that the experiment doesn’t have negative long term effects or interaction with other experiments.
```